File: MODUL 2 PEMROGRAMAN 1/PRAK203-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main(){
    //input 1// //input 2//
    float a, b, i, j, x, y, hsl;
    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &i);
    scanf("%f", &j);
    scanf("%f", &x);
    scanf("%f", &y);
    hsl = (a - b) * (i / j) - (x + y);
    printf("%.3f", hsl);
}

    

File: MODUL 2 PEMROGRAMAN 1/PRAK203-2310817120002-SITI RATNA DWINTA SARI.py
#Input 1
a, b, i, j, x, y = map(float, input ().split())
output = (a-b) * (i/j) - (x+y)
print(f"{output:.3f}")

#Input 2
a, b = map(float, input ().split())
i, j = map(float, input ().split())
x, y = map(float, input ().split())
output = (a-b) * (i/j) - (x+y)
print(f"{output:.3f}")